babel.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
             "@babel/preset-env",
             {
                 corejs: 3,
+                loose: true,
```

Comment:
Probably worth a comment here noting why we have this setting, since it seems important.

Reply:
Commented.